Gemfile.lock

```diff
@@ -1,258 +0,0 @@
-GEM
```

Comment:
Why are you deleting this file?

Reply:
At this branch we configured Gemfile by adding all gems that we need in our working process on this moment. Gemfile.lock will be generated automatically after bundle install on the basis of Gemfile. So we made decision to delete it and save time of every team participant to avoid resolving conflicts in Gemfile in future. The same situation is with schema.rb file. As shema.rb file looks like a snapshot database on some period of time it will be automatically generated after every rails db:migrate command. The second step we made is configuring of .rubocop.yml file. We configured file taking on account remarks of reviewers. As we had problems with creating migration on develop branch we made decision to make corrections to associations in approved models. Now this problem is solved.

Reply:
There is no need to remove this file. What about git ignore?

Reply:
Removing the files to avoid resolving the conflicts -- it is not a reason. You all have a problem with it and you should learn hot to do it. It is not so hard. 

what are the problems with migrations?